component/src/main/java/me/androidbox/component/ui/theme/Color.kt

```diff
@@ -0,0 +1,60 @@
+package me.androidbox.component.ui.theme
+
+
+import androidx.compose.material3.ColorScheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+val Purple200 = Color(0xFFBB86FC)
+val Purple500 = Color(0xFF6200EE)
+val Purple700 = Color(0xFF3700B3)
+val Teal200 = Color(0xFF03DAC5)
+val LightGray = Color(0xFFA1A4B2)
+val DarkGray = Color(0xFF5C5D5A)
+val ExtraLightGray = Color(0xFFF2F3F7)
+val LightBlue = Color(0xFFB7C6DE)
+val Red = Color(0xFFFF7272)
+val Black = Color(0xFF16161C)
+val LightWhite = Color(0xFFEEF6FF)
+
+val ColorScheme.placeholderEntry: Color
+    @Composable
+    get() {
+        return LightGray
+    }
```

Comment:
Couldn't this just be:
```
val ColorScheme.placeholderEntry = LightGray
```

Reply:
@MKiperszmid 
After applying that change I got the following error:
`Extension property cannot be initialized because it has no backing field`